src/EnergyPlus/CMakeLists.txt

```diff
@@ -1000,32 +1000,9 @@ if(BUILD_PACKAGE)
       install(FILES $<TARGET_FILE_DIR:energyplus>/IDFVersionUpdater.lnk DESTINATION "./" COMPONENT Auxiliary)
     endif()
     if(APPLE AND PYTHON_CLI)
-      add_custom_target(EPLaunchApp ALL
-              COMMAND ${CMAKE_COMMAND}
-              -D PROJECT_SOURCE_DIR=${PROJECT_SOURCE_DIR}
-              -D APP_NAME="EPLaunch"
-              -D ARGUMENT="eplaunch"
-              -D PRODUCTS_DIR=$<TARGET_FILE_DIR:energyplus>
-              -P ${PROJECT_SOURCE_DIR}/cmake/create_mac_bundle.cmake
-              COMMENT "Creating macOS app bundle for EPLaunch"
-      )
-      add_dependencies(EPLaunchApp energyplus)
-      install(DIRECTORY $<TARGET_FILE_DIR:energyplus>/EPLaunch.app
-              DESTINATION "."
-              USE_SOURCE_PERMISSIONS)
-      add_custom_target(VersionUpdaterApp ALL
-              COMMAND ${CMAKE_COMMAND}
-              -D PROJECT_SOURCE_DIR=${PROJECT_SOURCE_DIR}
-              -D APP_NAME="VersionUpdater"
-              -D ARGUMENT="updater"
-              -D PRODUCTS_DIR=$<TARGET_FILE_DIR:energyplus>
-              -P ${PROJECT_SOURCE_DIR}/cmake/create_mac_bundle.cmake
-              COMMENT "Creating macOS app bundle for VersionUpdater"
-      )
-      add_dependencies(VersionUpdaterApp energyplus)
-      install(DIRECTORY $<TARGET_FILE_DIR:energyplus>/VersionUpdater.app
-              DESTINATION "."
-              USE_SOURCE_PERMISSIONS)
+      # Pre-signed and notarized launcher apps
+      install(DIRECTORY "${PROJECT_SOURCE_DIR}/bin/PythonApps/EPLaunch.app" DESTINATION "." COMPONENT Auxiliary USE_SOURCE_PERMISSIONS)
+      install(DIRECTORY "${PROJECT_SOURCE_DIR}/bin/PythonApps/VersionUpdater.app" DESTINATION "." COMPONENT Auxiliary USE_SOURCE_PERMISSIONS)
```

Comment:
Just cmake install directory, with USE_SOURCE_PERMISSIONS so that the bash script is still chmod +x.

We should do the same for other .app bundles we have, so we can remove this

https://github.com/NREL/EnergyPlus/blob/9814fa5069b4cf52457ff0f9d18c7422f83eb33a/cmake/qtifw/install_operations.qs#L14-L17

And it would work with the TGZ too...

@Myoldmopar

Reply:
But really I think we should remove both of them now.